core/mondoo-aws-security.mql.yaml

```diff
@@ -2142,19 +2229,17 @@ queries:
   - uid: mondoo-aws-security-ebs-snapshot-public-restorable-check
     title: Ensure EBS snapshots are not publicly restorable
     impact: 80
-    filters: |
-      asset.platform == "aws-ec2-snapshot" || asset.platform == "aws-ebs-snapshot"
```

Comment:
same as above, some customers might be using this on this specific asset, there's no need to change the asset type.

Reply:
changed to variant, done